docs/cases/api/cases-api/cases-api-find-cases-by-alert.asciidoc

```diff
@@ -1,33 +1,33 @@
 [[cases-api-find-cases-by-alert]]
-=== Find Cases by Alert
+=== Find cases by alert
 
-Retrieves all case IDs that have a specific alert attached to the cases.
+Uses an alert ID (`alert id`) to retrieve the case IDs of any cases the alert is attached to.
 
-NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
+TIP: The case ID is identical to the case's saved object ID.
 
-NOTE: The case ID refers to the cases saved object ID field.
+NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
 ==== Request URL
 
-`GET <kibana host>:<port>/api/cases/alerts/<alert ID>`
+`GET <kibana host>:<port>/api/cases/alerts/<alert id>`
 
 ===== URL parts
 
-The URL must include the `alert ID` of the alert that you are interested in find the associated cases it is attached to.
+The URL must include the `alert ID` of the alert that is attached to the case.
```

Comment:
I think it should be something along the lines "of the alert that is attached to some/a case". The word "the case" makes me think that the alert should be attached to a specific case and not any case.

Reply:
Ah, good point. We definitely don't want to imply that the alert can only be attached to a single case. One alternative is to remove any mention of cases, for example, "The URL must include the `alert ID` of the attached alert." Another option is to rewrite the sentence to something like "The URL must include the alert ID (alert id), which is used to any cases the alert is attached to."

Reply:
Whatever works best for you 🙂 ! Both of them, lgtm.